csrc/multidevice/communication.cpp

```diff
@@ -203,7 +192,7 @@ c10::intrusive_ptr<c10d::Work> postBroadcast(
     }
   }
 
-  if (params.team.size() == 1) {
+  if (communication->team().size() == 1) {
```

Comment:
This was an intermediate change that turns out to be unnecessary for this PR. However, https://github.com/NVIDIA/Fuser/pull/2256 will remove CommParams, so I chose to keep this change which makes the code closer to the end state.